easybuild/easyconfigs/l/libcmaes/libcmaes-0.9.5-foss-2016a.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'libcmaes'
+version = '0.9.5'
+
+homepage = 'http://beniz.github.io/libcmaes/'
+description = """libcmaes is a multithreaded C++11 library for high performance
+blackbox stochastic optimization using the CMA-ES algorithm
+for Covariance Matrix Adaptation Evolution Strategy."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/beniz/%(namelower)s/archive']
+sources = ['%(version)s.tar.gz']
+
+dependencies = [
+    ('Eigen', '3.2.8'),
+]
+
+preconfigopts = "./autogen.sh && "
+
```

Comment:
please add custom `sanity_check_paths` (it helps a great deal in catching build problems that went under the radar):

```python
sanity_check_paths = {
    'files': ['lib/libcmaes.a', 'libcmaes.%s' % SHLIB_EXT, 'lib/pkgconfig/libcmaes.pc'],
    'dirs': ['bin', 'include/libcmaes'],
}
```